### ISSUE TITLE
add review criterion about community standards

### ISSUE THREAD

**Issue by danielskatz** [openjournals/joss#839]
I suggest we add a review criterion: 
* Does the software uses community/disciplinary standards for APIs and I/O, if appropriate.

**Comment by satra** [comment]
this should address both I and R in FAIR. 

a few additional things that could be useful is to use the review and submission process to create a structured component (jsonld snippet) for discovery that describes relevant components (hardware requirements, software requirements, standards, etc.,.)  for execution of software to be embedded in the repo in structured form. this is likely to evolve with the software, but having a structured view of this, rather than just a README may be useful for automated discoverability.

**Comment by danielskatz** [comment]
note that JOSS does not currently aim at FAIR, in particular since there is no community-accepted definition of what FAIR means for software.

**Comment by arfon** [comment]
> Does the software uses community/disciplinary standards for APIs and I/O, if appropriate.

What would be the guidance for reviewers on this point? Would it fail the review if the answer was 'no'?

**Comment by danielskatz** [comment]
Correct, the review would fail.  Reviewers (particularly in a discipline) should be able to make judgement.  For example, astronomy image processing software that writes our files in some new format that's not FITS would be flagged unless they explain why FITS isn't appropriate.
